packages/components/src/utils/theme-variables.scss

```diff
@@ -4,6 +4,10 @@
 // `--wp-admin-theme-color`. If the `--wp-admin-theme-color` variable is not
 // defined, fallback to the default theme color (WP blueberry).
 $components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
+
+// Define accent color transparent variants.
+$components-color-accent-transparent-40: rgba(var(--wp-components-color-accent--rgb, var(--wp-admin-theme-color--rgb)), 0.04);
+
```

Comment:
While reviewing `--wp-components-color-*` variables, @mirka and I came across this change that was added just before merging the PR and escaped our review.

- the `--wp-components-color-accent--rgb` custom property does not exist (we don't have any `--rgb` variants);
- the `$components-color-accent-transparent-40` SASS variable doesn't need to exist as a common variable, as it can be derived directly where it's needed via the `color-mix()` function
- it doesn't work on dark backgrounds (ie dark mode)

We should remove this variable and figure out an alternative solution that only makes use of existing variables and works with all possible theme combinations.